COC_PyBay_Reporting.md

```diff
@@ -0,0 +1,38 @@
+#Code of Conduct Reporting Guide
+
+In any PyBay space (events or online), if you believe someone is violating the Code of Conduct, we ask that you report it in one of the following ways:
```

Comment:
In person events or online events? It feels like "events or online" is a weird dichotomy.

Reply:
updated